server/app-engine/static/graas.js

```diff
@@ -921,6 +922,9 @@ function agencyIDCallback(response) {
         let arg = Date.now()
         util.log("- arg: " + arg);
         getURLContent(agencyID, arg);
+        // need to decide how to determine vehicle id
+        inf = await new TripInference('~/tmp/',agencyID, vehicleID, 15);
```

Comment:
The constructor isn't `async` and just assigns a few instance fields, why is there an `await`?